build/commands/lib/config.js

```diff
@@ -287,9 +287,15 @@ Config.prototype.buildArgs = function () {
     args.last_chrome_installer = this.last_chrome_installer
   }
 
+  if (process.platform === 'darwin') {
```

Comment:
@bridiver do we need to exclude iOS here?

Reply:
@mihaiplesa I believe we will need to, as iOS requires system Xcode: https://source.chromium.org/chromium/chromium/src/+/main:build/config/ios/ios_sdk.gni;l=13

Reply:
I think we're good since we reverted this https://github.com/brave/brave-core/pull/9116/files#diff-4a7ab71a64ac5712bb35fb91782847039cf2def6ee9c3d64a7efa66562a99acfR408